lua/apg/modules/ghosting.lua

```diff
@@ -47,7 +47,28 @@ function ENT:SetCollisionGroup( group )
 		group = COLLISION_GROUP_INTERACTIVE
 	end
 
-	return APG._SetColGroup( self, group )
+	return APG._SetCollisionGroup( self, group )
+end
+
+ENT._SetColor = ENT._SetColor or ENT.SetColor
+
+function ENT:SetColor( color, ... )
+	local color = color
+	local r, g, b, a
+
+	if type(color) == "number" then
+		color = Color(color, select(1, ...) or 255, select(2, ...) or 255, select(3, ...) or 255)

```

Comment:
Could you leave a comment explaining this line for future maintainers? Why was 255 selected as a default, what does the `select` do in this context, etc.

Reply:
`select` picks a variable from a list since `...` is of an undetermined size.
255 is the maximum colour value `Color(255, 255, 255, 255)` is pure white, and usually uncolours an entity.